lib/cylc/scheduler.py

```diff
@@ -697,10 +697,18 @@ def configure_suite(self, reconfigure=False):
 
             self.hold_suite_now = False
             self._pool_hold_point = None
+
+            if self.config.cfg['scheduling']['hold after point']:
+                self._pool_hold_point = get_point(
+                    self.config.cfg['scheduling']['hold after point'])
+
             if self.options.hold_point_string:
                 self._pool_hold_point = get_point(
                     self.options.hold_point_string)
 
+            if self._pool_hold_point:
+                print "[info] suite will hold after " + str(self._pool_hold_point)
```

Comment:
This stdout string prefix is not used in cylc.  I would like rationalize all stdout, stderr (along with logging) and consistently use "[info]", "[warn]", etc. ... but we haven't done that yet.  (Actually, there is one other instance put in recently, for "[info] copying across old suite database...")